ugh/src/ugh/fileformats/excel/Excelfile.java

```diff
@@ -2024,7 +2021,7 @@ public boolean CalculateEndPage(DocStruct inStruct)
 								// (physical struct).
 								List<Reference> refs = currentchild
 										.getAllReferences("to");
-								if (refs.size() == 0) {
+								if (refs == null || refs.size() == 0) {
 									// No references set, so set one to the
```

Comment:
IMHO that does not improve readability in this example. Is there a style guide which requires to use brackets here? Then much of the current code violates it. I found not a single line which uses the pattern which you suggest.